conda_build/main_build.py

```diff
@@ -255,6 +255,7 @@ def execute(args, parser):
     import shutil
     import tarfile
     import tempfile
+    from os import makedirs
```

Comment:
We do that a lot in conda, because os and os.path functions are used so ubiquitously. I know that this differs from common usage where only the module is imported. This way makes for much less verbose code that makes use of many os and os.path functions.